lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4886,8 +4882,8 @@ LogicalResult StmtEmitter::visitSV(CoverConcurrentOp op) {
 
 /// Emit an assert-like operation from the `verif` dialect. This covers
 /// `verif.assert`, `verif.assume`, and `verif.cover`.
-LogicalResult StmtEmitter::emitVerifAssertLike(Operation *op, Value property,
-                                               PPExtString opName) {
+template <typename Op>
```

Comment:
Does the op type actually matter here?  other than getProperty, I don't think it does.  In which case, taking an Operation* will result in smaller code.